libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -2822,6 +2839,19 @@ int WaveTrack::GetNumClips() const
    return mClips.size();
 }
 
+int WaveTrack::GetNumClips(double t0, double t1) const
+{
+   const auto clips = SortedClipArray();
+   const auto firstIn =
+      std::find_if(clips.begin(), clips.end(), [t0](const auto& clip) {
+         return t0 < clip->GetPlayEndTime();
+      });
+   const auto firstOut = std::find_if(firstIn, clips.end(), [t1](const auto& clip) {
```

Comment:
You might use `std::lower_bound` twicd instead of the linear searching `find_if`